image/equalize_adapthist_44312042/README.md

```diff
@@ -0,0 +1,9 @@
+# Contrast Limited Adaptive Histogram Equalization (CLAHE)
```

Comment:
Too minimal. Need info such as what it does, dependencies and example outputs/inputs

Reply:
Updated with more info in latest commit